src/main/java/hw6/core/entities/datafromjsonfile/Data1.java

```diff
@@ -0,0 +1,40 @@
+package hw6.core.entities.datafromjsonfile;
+
+import com.google.gson.JsonElement;
+
+public class Data1 {
+    protected static JsonElement summary;
```

Comment:
Class, that represents data element, should not consist of JsonElement(current abstraction layer should not be care about source of this data) filed and etcetera, only raw datatypes.